aries/aries_vcx_ledger/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "aries_vcx_ledger"
+version.workspace = true
+authors.workspace = true
+description.workspace = true
+license.workspace = true
+edition.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+vdr_proxy_ledger = ["dep:indy-vdr-proxy-client"]
```

Comment:
It would be ideal if we use `vdr_proxy_ledger`, we don't actually compile and bring in `indy-vdr` itself (as it brings/requires all the zmq stuff with it).

I suppose the reason why this can't be done now (having 2 separate feature `vdr_proxy_ledger` and `vdr_zmq_ledger`) is because even when you use proxy implementations, I guess we are still relying on some types defined indy `indy-vdr` (which carries the zmq stuff).

If you can confirm my suspition, there's potential refactoring task:
Refactor indy-vdr library such that it's broken down into
 - types crates
 - logic
 
 So that we can use the types, given that's necessary regardless of what transport we use, but not necessary require zmq library installed on the system if the user only intends to use `proxy` implementation.